doc/releases/release-notes-2.4.rst

```diff
@@ -303,6 +306,8 @@ Boards & SoC Support
 * Made these changes in other boards:
 
   * b_l072z_lrwan1: Added flash, LoRa, USB, EEPROM, RNG
+  * nRF boards: enabled HW Stack Protection by default on boards maintained by Nordic
+  * nRF5340 DK: added USB support
```

Comment:
This board was added in this release, so this entry looks a bit odd in the "changes in other boards" section.